lobster/tools/codebeamer/codebeamer.py

```diff
@@ -45,17 +45,18 @@
 import json
 import requests
 
-from lobster.items import Tracing_Tag, Requirement
+from lobster.items import Tracing_Tag, Requirement, Implementation, Activity
 from lobster.location import Codebeamer_Reference
 from lobster.errors import Message_Handler, LOBSTER_Error
 from lobster.io import lobster_read, lobster_write
 
 
 class References(Enum):
     REFS = "refs"
+    SCHEMA = "schema"
```

Comment:
Why did you add `SCHEMA` here?

Reply:
> Why did you add `SCHEMA` here?

If we don't provide the supported reference, it will throw a KeyError.